### ISSUE TITLE
Buttons are not rendering properly on windows

### ISSUE THREAD

**Issue by TransmissionZ** [kivymd/KivyMD#259]
### Description of the Bug

I made a simple app on windows to see if the button edges are sharp as the android demo but the edges are distorted and it doesn't look good.

The demo app "Kitchen_Sink" also has the same issue with buttons.

### Code and Logs

```python:

from kivy.uix.widget import Widget
from kivy.uix.screenmanager import ScreenManager, Screen
from kivy.uix.floatlayout import FloatLayout
from kivymd.app import MDApp
import kivy
from kivymd.uix.button import MDRoundFlatButton
import sys
import os

#from plyer import filechooser
from kivy.uix.image import Image


class HeaderImage(Image):
    def __init__(self, *args, **kwargs):
        Image.__init__(self, *args, **kwargs)



class MainWindow(Screen, Widget):

    def on_click(self, *args):
        #path = filechooser.open_file(title="Pick a show tech file",
        #                             filters=["*.txt", ".log"])
        sm.current = "second"
        #print(path)
        #return path
    pass

class MainMenuWindow(Screen):
    pass


def resourcePath():
    '''Returns path containing content - either locally or in pyinstaller tmp file'''
    if hasattr(sys, '_MEIPASS'):
        return os.path.join(sys._MEIPASS)

    return os.path.join(os.path.abspath("."))

class ParserApp(MDApp):
    def build(self):
        self.theme_cls.primary_palette = "Blue"  # "Purple", "Red"
        self.theme_cls.primary_hue = "A700"  # "500"
        self.theme_cls.theme_style = "Dark"
        global sm
        sm = ScreenManager()
        sm.add_widget(MainWindow(name = "main"))
        sm.add_widget(MainMenuWindow(name='mainmenu'))
        return sm


if __name__ == "__main__":
    kivy.resources.resource_add_path(resourcePath())
    ParserApp().run()


kv = """


<HeaderImage>:
    allow_stretch : True
    keep_ratio: True
    size_hint: 1,0.2
    size: 1000, 151
    width: 100
    pos_hint: {'x':0, 'top': 1}

<MainWindow>:
    name: "main"
    FloatLayout:
        orientation: "vertical"
        size : root.width, root.height

        HeaderImage:
            source: "Data\\main.png"


        MDFillRoundFlatButton:
            size_hint: .3, .05
            pos_hint: {'x':.35, 'y':.5}
            text: "Menu Driven Parsing"
            on_press: root.on_click()
            #on_release:
                #app.root.current = "mainmenu"

        MDFillRoundFlatButton:
            size_hint: .3, .05
            pos_hint: {'x':.35, 'y':.4}
            text: "Directory Driven Parsing"
            on_release:
                app.root.current = "mainmenu"

        MDFillRoundFlatButton:
            size_hint: .3, .05
            pos_hint: {'x':.35, 'y':.3}
            text: "About Us"
            on_release:
                app.root.current = "mainmenu"


<MainMenuWindow>:
    name: "mainmenu"
    FloatLayout:
        size: self.size
        pos: self.pos


        MDRectangleFlatButton:
            id : btn
            text: "Information CSV"
            size_hint: .2, .06
            pos_hint: {'x': 0, 'y':.6}

        MDRaisedButton:
            id : btn
            text: "Interface CSV"
            size_hint: .2, .05
            pos_hint: {'x': 0, 'y':.525}
            elevation_normal: 2
            opposite_colors: True
            pos_hint: {"center_x": .5}
            #md_bg_color: 1, 0, 0, 1

        MDFillRoundFlatButton:
            id : btn
            text: "Vlan CSV"
            size_hint: .2, .05
            pos_hint: {'x': 0, 'y':.45}

        MDRoundFlatButton:
            id: back
            text: "Go Back"
            size_hint: .3, .05
            pos_hint: {'x':.35, 'y':.1}
            width: dp(50)

            on_release:
                app.root.current = "main"

        MDFloatingActionButton:
            icon: "check"
            opposite_colors: True
            elevation_normal: 2


### Screenshots

![k](https://user-images.githubusercontent.com/49596652/79864899-ad89c400-83f3-11ea-830e-e95702f65a75.PNG)
![Capture](https://user-images.githubusercontent.com/49596652/79864902-af538780-83f3-11ea-8d2a-0a92caf41e03.PNG)



### Versions

* OS: Windows 10
* Python: 3.7.3
* Kivy: 1.11.1
* KivyMD: 0.104.0

**Comment by HeaTTheatR** [comment]
@TransmissionZ Correct your code. He does not work.

**Comment by HeaTTheatR** [comment]
@TransmissionZ In any case, I do not know what to answer, because we use `KivyMD`, including on `Windows`. I have never seen the behavior you describe here.

**Comment by TransmissionZ** [comment]
Fixed the code above! 

Yeah but the thing is, the kitchen_sink app buttons are also not sharp/clear.

From where can this issue arise?

Thank you for quick replies!

**Comment by HeaTTheatR** [comment]
@TransmissionZ I dont know. Maybe something with the drivers. How are Kivy widgets displayed on your computer?

**Comment by TransmissionZ** [comment]
![buttons](https://user-images.githubusercontent.com/49596652/79865252-4caebb80-83f4-11ea-96f0-4470f9b6931a.PNG)

The above image shows 5 buttons:
First two: Rectangle Buttons
middle one: Round Fill Button
Last 2: Normal Kivy Buttons. (Works fine)

Alright, I figured out rectangle buttons are working perfectly fine!

The round button edges are very un-sharp 👯‍♀️

**Comment by podraco** [comment]
This was a bug
The stencil mask created a Blury image in some configurations of the button.
It also created an unnecessary mipmapping that diffused the text texture
This is solved in #501
